package.json

```diff
@@ -32,9 +32,8 @@
   },
   "devDependencies": {
     "lodash": "^2.4.1",
-    "jstransform": "^6.2.0",
+    "jsx-loader": "0.13.2",
```

Comment:
It keeps pace with jstransform better.  (It's written by a Facebook alumnus and is commonly used for transpilation in the React community.)

It could also be worth looking into [babel](https://babeljs.io).  It's going to support more transformations.  Long term, I expect jstransform to be deprecated in favor of babel.